sdk/python/feast/infra/registry/sql.py

```diff
@@ -269,11 +286,31 @@ def _refresh_cached_registry_if_necessary(self):
                 logger.info("Registry cache expired, so refreshing")
                 self.refresh()
 
+    def _check_if_registry_refreshed(self):
+        CACHE_REFRESH_THRESHOLD_SECONDS = 300
```

Comment:
Maybe we should do this as a global variable so that we do not need to handle value inside the method and can control outside. In the future, we can make this as configurable value based on need.

Reply:
Fixed.